images/Dockerfile.py3

```diff
@@ -71,44 +77,20 @@ RUN export ASM_VERSION=1.4.7-asm.0 && \
     mv istio-${ASM_VERSION} /usr/local && \
     ln -sf /usr/local/istio-${ASM_VERSION}/bin/istioctl /usr/local/bin/istioctl
 
-    
-# Create go symlinks
-RUN ln -sf /usr/local/go/bin/go /usr/local/bin && \
-    ln -sf /usr/local/go/bin/gofmt /usr/local/bin && \
-    ln -sf /usr/local/go/bin/godoc /usr/local/bin
-
-RUN go get github.com/kelseyhightower/kube-rsa
-
-COPY ./images/checkout_repos.sh /usr/local/bin
-COPY ./images/checkout.sh /usr/local/bin
-COPY ./images/setup_ssh.sh /usr/local/bin
-RUN chmod a+x /usr/local/bin/checkout* /usr/local/bin/setup_ssh.sh
+COPY checkout_repos.sh /usr/local/bin/
```

Comment:
Ah, when building locally it doesn't work.

Reply:
Did you try setting your context to be the parent directory?

Reply:
I will try that, and will use Skaffold to test instead of just running locally. So Skaffold is a new dependency and wasn't used in the previous non-Python3 image, correct?